recipes/mpcmd/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: mpcmd
+  version: 0.0.5
+
+source:
+  url: https://files.pythonhosted.org/packages/bf/8c/07e22c42458f6dac37409979358b366cdf0c242bf648183cbdab6fba8eb5/mpcmd-0.0.5.tar.gz
```

Comment:
Please use the canonical url shown in the example `meta.yaml`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16